fms/models/gpt_bigcode.py

```diff
@@ -0,0 +1,324 @@
+import math
+from dataclasses import dataclass
+from typing import Optional, Tuple
+
+import torch
+import torch.nn as nn
+from fms.utils.config import ModelConfig
+from fms.modules.feedforward import FeedForwardBlock
+from fms.utils.activation import str_to_activation
+
+from fms.modules.attention import MultiHeadAttention
+
+
+@dataclass
+class GPTBigCodeConfig(ModelConfig):
+    src_vocab_size: int = 49280
+    emb_dim: int = 2048
+    emb_kq: Optional[int] = None
+    emb_v: Optional[int] = None
+    nheads: int = 12
+    kvheads: int = 1
+    nlayers: int = 12
+    pad_id: int = 0
+    max_pos: int = 512
+    vocab_bias: bool = False
```

Comment:
If we have configs that don't vary for "GPTBigCode" we might not want to make them configurable. I'm not sure which ones actually vary for different variants of GPTBigCode.

Reply:
Yes, I think we can maybe remove things like `vocab_bias` and `use_bias`

Reply:
updated in latest commit